src/containers/ExecuteButtons.js

```diff
@@ -6,9 +6,10 @@ import { getConnectionStatus } from '../actions';
 import ExecuteButton from '../components/ExecuteButton';
 
 export const ExecutePlaybookButton = withRouter(connect(
-    ({ connectionStatus: { data, status }}) => ({
+    ({ connectionStatus: { data, status }, selectedRemediation }) => ({
         data,
-        isLoading: status !== 'fulfilled'
+        isLoading: status !== 'fulfilled',
+        issueCount: selectedRemediation.remediation.issues.length
```

Comment:
seeing this reminds me about a idea we had a while back for better performance by returning the statistics instead of concrete  data from the api to avoid loading the concrete data from the database and passing it around

while this may not manifest before the summit, i believe it might still be helpful to line it up,
as i'm under the impression that with FiFi we will see re-mediations with larger issue counts more often
and its seems like a reasonable opportunity to keep things small/fast

i reckon that we need some more data to give this the right priority

Reply:
I  generally agree with you that it is better to use statistics instead of concrete data. However, this PR adds button that launches modal. At the screen where is the button placed are data in `selectedRemediation.remediation.issues` shown in a table. I don't create new request for issues here, I am just using data we requested earlier.

Even though we already have issue data, it would be better to just use count, unfortunately API doesn't return it. So the issue array is the only thing I can work with now.

Reply:
i agree, its just something i remembered while noting the data usage
with my current knowledge i believe its not critical to enable yet,
but its worth noting as something that could come up sooner than expected

after data collection i will open up relevant cards